src/pages/open source for non-developers.md

```diff
@@ -0,0 +1,23 @@
+# Open Source for Non-Developers
+
+## What is Open Source?
+
+Open source is software containing source code that is freely modified and redistributed. Unlike products that contain proprietary code, open source code can be viewed by developers worldwide. It can also allows people to propose changes, which fosters a collaborative working environment.
+
+## Why do Products Become Open Source?
+
+1. To interact with consumers: By making the designs and code open source, consumers have a more direct way to improve the products they enjoy, which in turn can save your company time and effort in creating troubleshooting guides.
+2. To reach more demographics: Open source products tend to gain their following through social media, so this aspect of these tools can help increase brand awareness.
+3. To build loyalty and trust: Since open source products allows people to see the source code. This form transparency help to build trust between consumers and the product’s creators.
```

Comment:
[Typo]

```suggestion
3. To build loyalty and trust: Since open source products allow people to see the source code, the transparency helps to build trust between consumers and the product’s creators.
```

[Praise] Really like these three points btw!